docs/how-to/field-configuration.md

```diff
@@ -105,3 +101,48 @@ class BlogPage(Page):
         SynchronizedField("slug"),
     ]
 ```
+
+## Specifying Translatable Fields within a StructBlock
+
+**By default, all sub-fields within a StructBlock are included for translation.** However, there may be instances where certain fields within the StructBlock should be excluded from translation. To facilitate this, we've introduced the **`translatable_blocks`** parameter.
```

Comment:
suggestion: wrap StructBlock, CharBlock, TextBlock and YouTubeBlock in backticks (so they are treated as inline code)